backend/core/src/seeder/listings-seeder/listings-seeder.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from "@nestjs/common"
+import { getConnection } from "typeorm"
+import listingsSeeds from "../../../seeds.json"
+import { Listing } from "../../entity/listing.entity"
+import { Unit } from "../../entity/unit.entity"
+import { Attachment } from "../../entity/attachment.entity"
+import { Preference } from "../../entity/preference.entity"
+
+@Injectable()
+export class ListingsSeederService {
+  async seed() {
+    const skipped = ["id", "units", "attachments", "preferences"]
+    const types = { units: Unit, attachments: Attachment, preferences: Preference }
+
+    const connection = getConnection()
+    const listings = listingsSeeds as any[]
+
+    for await (const listing of listings) {
+      const l = new Listing()
+      const entities = {}
+      for await (const key of Object.keys(listing)) {
```

Comment:
You can use `for (const <property> in <object>)` to iterate through objects.

Reply:
That code I was just copying and wrapping into a service, but OK I'll change that.

Reply:
Done